test/extensions/filters/http/lua/lua_integration_test.cc

```diff
@@ -445,7 +445,7 @@ name: envoy.lua
         request_handle:headers():add("signature_verification", "rejected")
       end
 
-      request_handle:releasePublicKey(pubkey)
+      request_handle:headers():add("verification", "done")
```

Comment:
Was this just ignored before?

Reply:
Yes. It was ignored before even thought we didn't define `releasePublicKey`.